open-api/rest-catalog-open-api.py

```diff
@@ -230,6 +230,10 @@ class BaseUpdate(BaseModel):
 
 
 class AssignUUIDUpdate(BaseUpdate):
+    """
+    Assigning a UUID to a table/view should only be done when creating the table/view. It is not safe to re-assign the UUID if a table/view already has a UUID assigned
```

Comment:
wondering if we expect the services to throw some validation exp or BadRequest in case there already exists a table and if yes should we document this here ?

Reply:
@singhpk234 this is being handled by the `TableRequirement` section, more specifically, there is `assert-create`: https://github.com/apache/iceberg/blob/621d301b17417af4b71a0e8c93db9db6f8266839/open-api/rest-catalog-open-api.yaml#L1831